sdk/keyvault/azure-security-keyvault-certificates/samples/certificate-basic-operations/certificate-basic-operations.cpp

```diff
@@ -66,7 +66,7 @@ int main()
     // create a certificate
     {
       // start the create process
-      auto response = certificateClient.StartCreateCertificate(certificateName, options);
+      auto response = certificateClient.StartCreateCertificate(options);
```

Comment:
I don't see `certificateName` as part of the options...  How are we sending the name?

Reply:
it's in the certificate properties, inside the options

Reply:
that's what the archboard wanted